src/web/client/WebExtensionContext.ts

```diff
@@ -82,6 +83,7 @@ class WebExtensionContext implements IWebExtensionContext {
     private _vscodeWorkspaceState: Map<string, IEntityInfo>;
     private _languageIdCodeMap: Map<string, string>;
     private _portalLanguageIdCodeMap: Map<string, string>;
+    private _sharedWorkSpaceMap: Map<string, string>;
```

Comment:
nit: lets keep copresence needed parameters in webExtensionContext in single place and add a comment on top, something like "// Copresence for Power pages"